packages/flutter_tools/lib/src/ios/fallback_discovery.dart

```diff
@@ -82,50 +82,50 @@ class FallbackDiscovery {
     }
 
     try {
-      final Uri result = await _mDnsObservatoryDiscovery.getObservatoryUri(
-        packageId,
-        device,
-        usesIpv6: usesIpv6,
-        hostVmservicePort: hostVmservicePort,
-      );
+      final Uri result = await _protocolDiscovery.uri;
```

Comment:
The change in this method is switching the order of fallbacks, which confused `git`.  Now it first tries log scanning, and only if that fails, falls back to mDNS discovery.